src/App.jsx

```diff
@@ -1,33 +1,22 @@
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import SignInPage from './pages/SignInPage';
-import GetStartedPage from './pages/GetStarted';
-import Home from './pages/Home';
-import PrivateRoute from './components/PrivateRoute';
-import CreateProject from './pages/CreateProject';
+import { BrowserRouter as Router, Routes, Route, RouterProvider } from 'react-router-dom';
```

Comment:
Remove unused variables, for the future please install the ESLint extension in your VS Code so that you can notify if you've any unused variables in your file.